pht_federated/aggregator/services/discovery/object_list_transform.py

```diff
@@ -75,22 +76,36 @@ def process_column(column_information: list, object_list_int: list) -> list:
 
     return object_list_int
 
+
 def retrieve_column(column_information: list, dataset_statistics_list: list) -> list:
 
     for column in column_information:
         if column["type"] == "categorical":
-            column, dataset_statistics_list = retrieve_categorical_column(column, dataset_statistics_list)
+            column, dataset_statistics_list = retrieve_categorical_column(
+                column, dataset_statistics_list
+            )
         elif column["type"] == "numeric":
-            column, dataset_statistics_list = retrieve_numeric_column(column, dataset_statistics_list)
+            column, dataset_statistics_list = retrieve_numeric_column(
+                column, dataset_statistics_list
+            )
         elif column["type"] == "unique":
-            column, dataset_statistics_list = retrieve_unique_column(column, dataset_statistics_list)
+            column, dataset_statistics_list = retrieve_unique_column(
+                column, dataset_statistics_list
+            )
         elif column["type"] == "equal":
-            column, dataset_statistics_list = retrieve_equal_column(column, dataset_statistics_list)
+            column, dataset_statistics_list = retrieve_equal_column(
+                column, dataset_statistics_list
+            )
         elif column["type"] == "unstructured":
-            column, dataset_statistics_list = retrieve_unstructured_column(column, dataset_statistics_list)
+            column, dataset_statistics_list = retrieve_unstructured_column(
+                column, dataset_statistics_list
+            )
```

Comment:
Seems like dataset_statistics_list is never used: should either be hidden (assign to variable `_` ) or simply not returned at all if not needed elsewhere.

Reply:
It is used in the respective "retrieve function" list elements are popped and new list without popped element is returned again

Reply:
but _dataset_statistics_list_ is modified inside the function -> function returns subset of input list